utilities/hc-leipzig-7t-mp2rage/get_statistics_hc-leipzig-7t-mp2rage.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import os
+import re
+import numpy as np
+
+'''
+This script computes metrics for nnUNetv2 results and saves them as csv files wit usage of MetricsReloader package.
```

Comment:
Same comments as for the previous script: change `'''` to `"""`, move imports below the description

Reply:
```
This script computes metrics for nnUNetv2 results and saves them as csv files wit usage of MetricsReloader package.
```
-->
```
This script computes segmentation metrics (Dice, ...) using the MetricsReloaded package for nnUNetv2 results and saves them as CSV files. One CSV file is generated for each GT-prediction pair.
```